### PR TITLE
Fix continuous integration

### PR DESCRIPTION
**Pull request by yvanzo**
* Fix path to built style sheet
* Rework test to provide more details on test failure
* Fix test preinstall to follow test setup
* Clean output of curl and docker-compose
* Run test on [`mbvm-38-dev`](https://github.com/metabrainz/musicbrainz-docker/tree/mbvm-38-dev) branch

Demo: [Demo build status](https://travis-ci.org/github/metabrainz/musicbrainz-docker/builds/677561256)

### CODE REVIEW

File: test/test-musicbrainz-image.sh [hunk 1]
```diff
@@ -1,14 +1,19 @@
 #!/bin/bash
 
-set -e -u
+set -e -o pipefail -u
```
Comment (SuperSandro2000) [inline test/test-musicbrainz-image.sh:3]:
```suggestion
set -eou pipefail
```

**Review by SuperSandro2000 — COMMENTED** [review]
GitHubs required comment to leave a comment

File: test/test-musicbrainz-image.sh (hunk 1 above)
Comment (yvanzo) [reply test/test-musicbrainz-image.sh:3]: Thanks but it doesn't make options more readable, it doesn't follow `set` specification either. Actually, `-e` and `-u` might even be replaced with `-o <option-name>` which is more explicit.

**Comment by yvanzo** [comment]
Completed with https://github.com/metabrainz/musicbrainz-docker/commit/ef5b1617329afcd3d1e354a8d4f0b23fbeeedd51 to also retry downloads as needed.
Finally make test to pass with [`mbvm-38-dev`](https://github.com/metabrainz/musicbrainz-docker/tree/mbvm-38-dev#readme) branch.

File: test/test-musicbrainz-image.sh (hunk 1 above)
Comment (SuperSandro2000) [reply test/test-musicbrainz-image.sh:3]: Being more explicit is not always good cause you need to read more to get the same context.

**Pull request merged**